fahrschein/src/main/java/org/zalando/fahrschein/domain/Authorization.java

```diff
@@ -0,0 +1,65 @@
+package org.zalando.fahrschein.domain;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Collections.unmodifiableList;
+
+@Immutable
+public final class Authorization {
+
+    @Immutable
+    public static final class AuthorizationAttribute {
+
+        public static final AuthorizationAttribute ANYONE = new AuthorizationAttribute("*", "*");
+
+        private final String dataType;
+        private final String value;
+
+        public AuthorizationAttribute(String dataType, String value) {
+            this.dataType = dataType;
+            this.value = value;
+        }
+
+        public String getDataType() {
+            return dataType;
+        }
+
+        public String getValue() {
+            return value;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            if (this == o)
+                return true;
+            if (!(o instanceof AuthorizationAttribute))
+                return false;
+            AuthorizationAttribute that = (AuthorizationAttribute) o;
+            return dataType.equals(that.dataType) &&
+                    value.equals(that.value);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(dataType, value);
+        }
+    }
+
+    private final List<AuthorizationAttribute> admins;
+    private final List<AuthorizationAttribute> readers;
+
+    public Authorization(List<AuthorizationAttribute> admins, List<AuthorizationAttribute> readers) {
+        this.admins = unmodifiableList(admins);
```

Comment:
This just ensure that `this.admins` can't be modified later. But the originally passed list `admins` could still change.

Reply:
True, but I don't see any problem about it. This is how it's done in the rest of the project, I don't really wanna change the approach for one DTO only.

Reply:
Should be fine here since the builder always creates new lists, so it's impossible to modify it from outside the library